build/dev/publish-devfile-registry-to-surge.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# Copyright (c) 2018-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+set -e
+
+DEFAULT_BUILD_DIR="/projects/build"
+BUILD_DIR=${BUILD_DIR:-$DEFAULT_BUILD_DIR}
+
+DEFAULT_SURGE_DIR="/projects/surge"
+SURGE_DIR=${SURGE_DIR:-$DEFAULT_SURGE_DIR}
+
+DEFAULT_REPO_DIR="/projects/che-devfile-registry"
+REPO_DIR=${REPO_DIR:-$DEFAULT_REPO_DIR}
+
+rm "$SURGE_DIR" -rf;
+mkdir -p "$SURGE_DIR"/images
+cd "$SURGE_DIR"
+echo '*' > CORS
+cp -rf "$BUILD_DIR/devfiles" "$SURGE_DIR/devfiles"
+cp -rf "$REPO_DIR"/images/* "$SURGE_DIR/images"
+
+CHE_DEVFILE_REGISTRY_URL="https://$CHE_WORKSPACE_NAMESPACE-$CHE_WORKSPACE_NAME.surge.sh"
```

Comment:
just a FYI but these env variables won't exist in dev workspace mode

Reply:
ok to rework that in https://github.com/eclipse/che/issues/19305 ?